<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchOrderWin</class>
 <widget class="QWidget" name="purchOrderWin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyOfSale - Purchase Order</string>
  </property>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantSpinbox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 121, 0);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search product:</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Total: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="includeBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Include</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
   <property name="icon">
    <iconset theme="checkbox">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="alphaOrderCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alphabetical Order</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplierLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Profit Margin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>35</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="salePrice">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.990000000005239</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sale Price:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="profitMargin">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>supplierLineEdit</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>quantSpinbox</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>salePrice</tabstop>
  <tabstop>includeBtn</tabstop>
  <tabstop>deleteBtn</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>finButton</tabstop>
  <tabstop>alphaOrderCheck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>quantSpinbox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quantSpinbox</sender>
   <signal>editingFinished()</signal>
   <receiver>doubleSpinBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>supplierLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>salePrice</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
